scripts/Calibration/tube.py

```diff
@@ -4,7 +4,8 @@
 Definition of Calibration
 =========================
 
-.. autofunction:: calibrate(ws, tubeSet, knownPositions, funcForm, [fitPar, margin, rangeList, calibTable, plotTube, excludeShorTubes, overridePeaks, fitPolyn, outputPeak])
```

Comment:
Unfortunately this breaks one of things I was attempting to fix. `autofunction` doesn't seem to like the signature split across multiple lines